draft-vanbrouwershaven-acme-auto-discovery.md

```diff
@@ -77,6 +81,24 @@ informative:
   RFC8657:
   RFC5785:
   I-D.tweedale-acme-discovery:
+  EVBRs:
+    title: "Minimum Security Requirements for Issuance of Mark Certificates, v2.0.1"
+    target: "https://cabforum.org/working-groups/server/extended-validation/documents/CA-Browser-Forum-EV-Guidelines-2.0.1.pdf"
+    author:
+      - org: CA/Browser Forum
+    date: 6 May, 2024
+  QWAC-ENISA:
+    title: "Qualified Website Authentication Certificates"
+    target: "https://www.enisa.europa.eu/publications/qualified-website-authentication-certificates"
+    author:
+      - org: European Union Aguncy for Cybersecurity (ENISA)
+    date: May 16, 2016
+  VMCRequirements:
+    title: "Guidelines for the Issuance and Management of Extended Validation Certificates"
+    target: "https://bimigroup.org/resources/VMC_Requirements_latest.pdf"
+    author:
+      - org: BIMI Group
+    date: March 7, 2024
```

Comment:
Do we really need to link to this information and do we want to include links to versions that will be out of date even before publication of the draft itself?

If we want to keep this information, I have correct the information and links to the current latest version.

```suggestion
  BR:
    title: "Baseline Requirements for the Issuance and Management of Publicly‐Trusted TLS Server Certificates"
    target: "https://cabforum.org/working-groups/server/baseline-requirements/requirements/"
    author:
      - org: CA/Browser Forum
  EVG:
    title: "Guidelines for the Issuance and Management of Extended Validation Certificates"
    target: "https://cabforum.org/working-groups/server/extended-validation/guidelines/"
    author:
      - org: CA/Browser Forum
  QWAC:
    title: "Electronic Signatures and Infrastructures (ESI); Policy and security requirements for Trust Service Providers issuing certificates; Part 2: Requirements for trust service providers issuing EU qualified certificates"
    target: "https://www.etsi.org/deliver/etsi_en/319400_319499/31941102/"
    author:
      - org: ETSI
  VMC:
    title: "Guidelines for the Issuance and Management of Extended Validation Certificates"
    target: "https://bimigroup.org/resources/VMC_Requirements_latest.pdf"
    author:
      - org: BIMI Group
```

Reply:
Agreed